src/sentry/web/client_config.py

```diff
@@ -236,7 +231,17 @@ def get_client_config(request=None):
     }
     if user and user.is_authenticated:
         context.update(
-            {"isAuthenticated": True, "user": serialize(user, user, DetailedSelfUserSerializer())}
+            {
+                "isAuthenticated": True,
+                "user": user_service.serialize_users(
+                    [user.id],
+                    detailed=2,
```

Comment:
Should we have an Enum for the various kinds of 'detailed' that users have? Or like Mike suggested pass the serializer implementation directly.

Reply:
Yup!  Addressed.